notebooks/week2/01.prepare_dataset.py

```
@@ -0,0 +1,54 @@
+
+
+# COMMAND ----------
+
+# Load project configuration from YAML file
+config = ProjectConfig.from_yaml(config_path="../../project_config.yml")
+
+# COMMAND ----------
+
+# Load the child health dataset into a Pandas DataFrame
+df = spark.read.csv(
+    "/Volumes/mlops_students/javedhassi/data/childHealth.csv", 
```

Ruff

Trailing whitespace

(W291)